understanding/about.html

```diff
@@ -168,87 +153,79 @@ <h2>About WCAG</h2>
             <section id="introduction">
                
                
-               <h2>About the Understanding Documents</h2>
+               <h2>解説書について</h2>
                
                
-               <p>WCAG Understanding documents are guides to understanding and implementing WCAG.  
-                  They provide detailed  explanations for
-                  each guideline and each success criterion to help readers better understand the intent
-                  of the success criteria. They include  background information and  technical details.
-                  They include <em>techniques</em> that are examples of ways to meet the success criteria. Each technique is linked
-                  to more details in a techniques page. Techniques are explained in <a class="understanding" href="understanding-techniques">Understanding Techniques for WCAG Success Criteria</a>.
-                  
+               <p>WCAG 解説書は、WCAG を理解して実践するための案内書である。各ガイドライン及び各達成基準の詳細な説明を提供しており、読者が達成基準の意図をよりよく理解できるよう支援するものである。その内容には、背景情報や技術的な詳細情報が含まれる。達成基準の満たしかたの例としての<em>テクニック</em>も含まれる。なお各テクニックは、テクニックの詳細ページにリンクされている。テクニックについての説明は、<a class="understanding" href="understanding-techniques">WCAG 達成基準のテクニックを理解する</a>を参照のこと。
                </p>
                
                
-               <p>WCAG Understanding documents are not  introductory resources. They are  for people
-                  who want to understand WCAG more thoroughly. The <a href="https://www.w3.org/WAI/intro/wcag">WCAG Overview</a> provides introductory information.
+               <p>WCAG 解説書は、入門的な資料ではない。WCAG をより深く理解したい人のためのものである。入門的な情報は、<a href="https://www.w3.org/WAI/intro/wcag">WCAG Overview</a> で提供されている。
                   
                </p>
                
                
                <section id="structure">
                   
                   
-                  <h3>Structure of the Documents</h3>
+                  <h3>解説書の構造</h3>
                   
                   
-                  <p><em>Understanding <strong>Guideline</strong> </em>pages include:
+                  <p><em><strong>ガイドライン</strong>の解説</em>ページには、以下が含まれる:
                      
                   </p>
                   
                   
                   <ul>
                      
                      
-                     <li>The intent</li>
+                     <li>意図</li>
                      
                      
-                     <li>Any advisory techniques that are related
-                        to the guideline and not specifically related to any of its success criteria
+                     <li>ガイドラインに関連する (その下の達成基準には特に関連しない) 参考テクニック
```

Comment:
<https://waic.jp/translations/WCAG22/Understanding/seizures-and-physical-reactions.html>
みたいなのを説明してると思いますが…とりあえずは、かっこ書きにしているわけでもないので、こうするのはどうでしょう。

```suggestion
                     <li>ガイドラインに関連し、その下位の達成基準には特に関連しない参考テクニック
```